test_scripts/API/VehicleData/StabilityControlsStatus/004_OnVehicleData_StabilityControlsStatus.lua

```diff
@@ -0,0 +1,114 @@
+-- SDL does:
+--  - not send OnVehicleData notification with received from HMI data to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+-- local common = require('test_scripts/API/VehicleData/StabilityControlsStatus/commonVDStabilityControlsStatus')
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+local function getVDParams()
+  local out = {}
+  for k in pairs(common.allVehicleData) do
+    table.insert(out, k)
+  end
+  return out
+end
+
+local function ptUpdate(pTbl)
+  pTbl.policy_table.app_policies[common.getConfigAppParams(2).fullAppID].groups = { "Base-4", "Emergency-1" }
+  pTbl.policy_table.app_policies[common.getConfigAppParams(1).fullAppID].groups = { "Base-4", "Emergency-1" }
+
+  local grp = pTbl.policy_table.functional_groupings["Emergency-1"]
+  local vdParams = getVDParams()
+  for _, v in pairs(grp.rpcs) do
+    v.parameters = common.cloneTable(vdParams)
+  end
+  pTbl.policy_table.vehicle_data = nil
+end
+
+local function processRPCSubscriptionSuccess(pRpcName, pData, pAppId, pIsCached)
+  local reqParams = {
+    [pData] = true
+  }
+  local respData
+  if pData == "clusterModeStatus" then
+    respData = "clusterModes"
+  else
+    respData = pData
+  end
+  local hmiResParams = {
+    [respData] = {
+      resultCode = "SUCCESS",
+      dataType = common.allVehicleData[pData].type
+    }
+  }
+  local cid = common.getMobileSession(pAppId):SendRPC(pRpcName, reqParams)
+  if not pIsCached then
+    common.getHMIConnection():ExpectRequest("VehicleInfo." .. pRpcName, reqParams)
+    :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", hmiResParams)
+      end)
+  end
+  local mobResParams = common.cloneTable(hmiResParams)
+  mobResParams.success = true
+  mobResParams.resultCode = "SUCCESS"
+  common.getMobileSession(pAppId):ExpectResponse(cid, mobResParams)
+end
+
+local function checkNotificationSuccess(pData, pApps)
+  local hmiNotParams = { [pData] = common.allVehicleData[pData].value }
+  local mobNotParams = common.cloneTable(hmiNotParams)
+  for _, appInfo in pairs(pApps) do
+    if appInfo.isNotified then
+      common.getMobileSession(appInfo.id):ExpectNotification("OnVehicleData", mobNotParams)
+    else
+      common.getMobileSession(appInfo.id):ExpectNotification("OnVehicleData"):Times(0)
+    end
+  end
+  common.getHMIConnection():SendNotification("VehicleInfo.OnVehicleData", hmiNotParams)
+
+end
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+-- common.Step("Prepare preloaded policy table", common.preparePreloadedPT)
+common.Step("Start SDL, init HMI, connect default mobile", common.start)
+common.Step("Register App", common.registerApp)
+common.Step("PTU", common.policyTableUpdate)
+common.Step("Register App", common.registerApp, { 2 })
```

Comment:
```suggestion
common.Step("Register App1", common.registerApp, { 2 })
```
I would propose to specify app and in the other steps below

Reply:
@VjKlepikov Fixed (see a7a20fe)